shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -670,8 +670,6 @@ async fn request_throttling(#[case] driver: CassandraDriver) {
 
     tokio::time::sleep(std::time::Duration::from_secs(1)).await; // sleep to reset the window
 
-    batch_statements::test(&connection).await;
```

Comment:
The new `large_batch` test in this module triggers the request throttling so I've removed it. We are already testing that the rate limiter tests batches both below and above the threshold so it should be fine to remove.